docs/tutorials/basics-docker.rst

```diff
@@ -27,11 +27,11 @@ Download the Tutorial and Start KSQL
        $ git clone git@github.com:confluentinc/ksql.git
        $ cd ksql
 
-#. Switch to the correct Confluent Platform release branch:
+#. Switch to the correct |cp| release branch:
 
    .. code:: bash
 
-       $ git checkout 4.1.0-post
+       $ git checkout |release_post_branch|
```

Comment:
within code blocks, you need to switch to `codewithvars` instead of just plain `code` or `sourcecode`

Reply:
done